package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "protractor-sync",
-  "version": "6.0.2",
+  "version": "6.0.3",
   "description": "Protractor 5.x wrapper adding synchronous, polled selectors and expectations",
```

Comment:
We might update this description to remove the version

Reply:
Hm. Why?

Reply:
Hah completely misread this, I thought it said Protractor Sync 5.x and was referencing itself.